pkg/tests/schema_pulumi_test.go

```diff
@@ -3019,14 +3014,8 @@ Resources:
         [id=newid]
         [urn=urn:pulumi:test::test::prov:index/test:Test::mainRes]
       ~ setBlocks: [
-          ~ [0]: {
-                  ~ prop: "val2" => "val1"
-                }
-          ~ [1]: {
-                  ~ prop: "val3" => "val2"
-                }
-          + [2]: {
-                  + prop      : "val3"
+          + [0]: {
```

Comment:
I must say these tests cases are very pleasant to read. These changes look convincing! We are getting things right.